test/prototype_transforms_kernel_infos.py

```diff
@@ -129,6 +129,9 @@ def sample_inputs_horizontal_flip_mask():
 
 def _get_resize_sizes(image_size):
     height, width = image_size
+    length = max(image_size)
+    # yield length
+    yield [length]
     yield height, width
```

Comment:
This will yield a tuple instead of a list. Currently my tests fail because of that. Should we patch it @pmeier?

Reply:
Not sure if we can. For all the input types that accept a `List`, I also added tuples as input arguments. This works fine from Python. We need to check if that is also possible from C++. If yes, we shouldn't remove.

Reply:
I just need a way to work around this on the other PR to check JIT-scriptability for lists. A temporary hack will work, even on the actual test kernel invocation side.

Reply:
As discussed offline, this PR is doing what @datumbox wants, i.e. adding sample inputs for everything that currently works.